.claude/skills/pr-workflow/SKILL.md

```
@@ -6,88 +6,25 @@ allowed-tools: Bash, Read, Glob
+dotnet build src/ZChain.sln    # warnings are errors, so this is a real gate
 dotnet test src/ZChain.sln
-
-# Check what changed
-git status
-git diff
 ```
 
-### Commit Message Format
+Commit messages are imperative mood, ~50-char summary, body explaining **why** rather than what, and:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)